components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -769,24 +769,44 @@ def start_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts
     logger.info(f"Started {component_name}.")
 
 
-def start_log_viewer_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts):
+def start_log_viewer_webui(
+    instance_id: str,
+    clp_config: CLPConfig,
+    container_clp_config: CLPConfig,
+    mounts: CLPDockerMounts,
+):
     component_name = LOG_VIEWER_WEBUI_COMPONENT_NAME
     logger.info(f"Starting {component_name}...")
 
     container_name = f"clp-{component_name}-{instance_id}"
     if container_exists(container_name):
         return
 
-    log_viewer_webui_logs_dir = clp_config.logs_directory / component_name
-    container_log_viewer_dir = CONTAINER_CLP_HOME / "var" / "www" / "log_viewer"
-    node_path = str(container_log_viewer_dir / "server" / "node_modules")
-
-    validate_log_viewer_config(clp_config, log_viewer_webui_logs_dir)
+    container_log_viewer_webui_dir = CONTAINER_CLP_HOME / "var" / "www" / "log_viewer_webui"
+    node_path = str(container_log_viewer_webui_dir / "server" / "node_modules")
+    settings_json_path = (
+        get_clp_home() / "var" / "www" / "log_viewer_webui" / "server" / "settings.json"
+    )
 
-    # Create directories
-    log_viewer_webui_logs_dir.mkdir(exist_ok=True, parents=True)
+    validate_log_viewer_webui_config(clp_config, settings_json_path)
 
-    container_log_viewer_webui_logs_dir = pathlib.Path("/") / "var" / "log" / component_name
+    with open(settings_json_path, "r") as settings_json_file:
+        settings_json = json.loads(settings_json_file.read())
+    settings_json_updates = {
+        "SqlDbHost": clp_config.database.host,
```

Comment:
I think you are overwriting every fields in the settings.json. in that case, do we still need to require the file to exist?

Reply:
I remember discussing a similar situation with @kirkrodrigues when we added the meteor settings.json override but I forgot the conclusion. Do you remember?

Reply:
The idea is that `settings.json` is an authoritative list of the settings that can exist. If we try to set a setting that doesn't exist, `update_settings_json` will complain.

Reply:
To make that clearer, perhaps we should load `settings.json` inside `update_settings_json`.